tidb-cloud/manage-serverless-spend-limit.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Manage Spend Limit for Serverless Tier clusters
+summary: Learn how to manage spend limit for your Serverless Tier clusters.
+---
+
+# Manage Spend Limit for Serverless Tier Clusters
+
+> **Note:**
+>
+> The spend limit is only applicable to Serverless Tier clusters.
+
+Spend limit refers to the maximum amount of money that you are willing to spend on a particular workload in a month. It is a cost-control mechanism that allows you to set a budget for your Serverless Tier clusters.
+
+For each organization in TiDB Cloud, you can create a maximum of five Serverless Tier clusters by default. To create more Serverless Tier clusters, you need to add a credit card and set a spend limit for the usage. But if you delete some of your previous clusters before creating more, the new cluster can still be created without a credit card.
+
+## Usage quota
+
+For each Serverless Tier cluster, TiDB Cloud provides a free usage quota as follows:
```

Comment:
Only maximum five of serverless clusters have free usage quota.

Reply:
updated in https://github.com/pingcap/docs/pull/13335/commits/19d6207a48bdc66b6aeb6d280fa9bad773aa2ce0